airflow/www/static/connection_form.js

```diff
@@ -7,7 +7,12 @@
         jdbc: {
             hidden_fields: ['port', 'schema', 'extra'],
             relabeling: {'host': 'Connection URL'},
+        },
+        google_cloud_platform: {
```

Comment:
Not sure if this will work with data profiler. Right now, we have different types for BQ, GCS, datastore, etc. This allows us to set data profiler to query BigQuery. This code is in models.py's get_hook method:

            elif self.conn_type == 'bigquery':
                return contrib_hooks.BigQueryHook(bigquery_conn_id=self.conn_id)

I suppose we could assume it's a BQ hook if the connection is of type 'google_cloud_platform'. But it's worth considering.